pyproject.toml

```diff
@@ -198,6 +198,8 @@ filterwarnings = [
     "ignore:'cgi' is deprecated and slated for removal in Python 3.13:DeprecationWarning",
     # Ignore numpy 2.0 warning, see https://github.com/astropy/astropy/pull/15495
     "ignore:numpy.core._multiarray_umath is deprecated:DeprecationWarning",
+    # Ignore numpy 2.0 warning, see https://github.com/scipy/scipy/pull/19275
+    "ignore:.*`numpy.core` has been made officially private.*:DeprecationWarning",
```

Comment:
Hmm, @bsipocz is right: probably good to add `:scipy` at the end here!

Reply:
I think I ignored the wrong thing...

DeprecationWarning: numpy.core.numeric is deprecated and has been renamed to numpy._core.numeric. The numpy._core namespace contains private NumPy internals and its use is discouraged, as NumPy internals can change without warning in any release. In practice, most real-world usage of numpy.core is to access functionality in the public NumPy API. If that is the case, use the public NumPy API. If not, you are using NumPy internals. If you would still like to access an internal attribute, use numpy._core.numeric.normalize_axis_tuple.

Reply:
Let's get it passing first before worrying about specifying module. Specifying module does not work for me most of the time anyway. The order of imports and how pytest is trying to catch it is all very confusing.